lib/mayaUsd/ufe/wrapUtils.cpp

```diff
@@ -213,6 +225,7 @@ void wrapUtils()
     def("usdPathToUfePathSegment",
         usdPathToUfePathSegment,
         (arg("usdPath"), arg("instanceIndex") = UsdImagingDelegate::ALL_INSTANCES));
+    def("getTime", getTime);
```

Comment:
Reading transform op values directly through the USD interface requires time (whereas the UFE interface uses proxy time).  Expose the time accessor in Python.